### PR TITLE
S05 us67

### PR DESCRIPTION
**Pull request by cnd24**

### CODE REVIEW

**Review by Toua45 — APPROVED** [review]

File: src/Controller/CompagnyController.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * @Route("compagny")
+ */
+class CompagnyController extends AbstractController
```
Comment (sblondeau) [inline src/Controller/CompagnyController.php]: sorry mais c'est company en english :/

**Review by sblondeau — CHANGES_REQUESTED** [review]

File: src/Controller/CompagnyController.php [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+     */
+    public function showCGU()
+    {
+        return $this->render('compagny/cgu_index.html.twig');
+    }
+
+    /**
+     * @Route("/mentionslegales", name="compagny_mentions_legales")
+     */
+    public function showMentionsLegales()
```
Comment (sblondeau) [inline src/Controller/CompagnyController.php]: legalMentions

File: src/Controller/CompagnyController.php [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    {
+        return $this->render('compagny/mentions_legales_index.html.twig');
+    }
+
+    /**
+     * @Route("/politiquedeconfidentialite", name="compagny_confidentialite")
+     */
+    public function showConfidentialite()
```
Comment (sblondeau) [inline src/Controller/CompagnyController.php]: showConfidentiality

**Review by sblondeau — CHANGES_REQUESTED** [review]

File: src/Controller/CompagnyController.php (hunk 1 above)
Comment (cnd24) [reply src/Controller/CompagnyController.php]: mince :-/

**Review by Toua45 — APPROVED** [review]

**Review by sblondeau — APPROVED** [review]

**Pull request merged**